tests/public/ts/models/ioobjects/WirePort.test.ts

```diff
@@ -0,0 +1,24 @@
+import "jest";
+
+import {CircuitDesigner} from "../../../../../site/public/ts/models/CircuitDesigner";
+import {WirePort}   from "../../../../../site/public/ts/models/ioobjects/WirePort";
+import {Switch}   from "../../../../../site/public/ts/models/ioobjects/inputs/Switch";
+import {LED}             from "../../../../../site/public/ts/models/ioobjects/outputs/LED";
+
+describe("Constant Low", () => {
```

Comment:
Describe should describe the component you're testing

Reply:
fixed